src/modules/AboutMe/styles.module.scss

```diff
@@ -6,9 +6,6 @@
 }
 
 .personalPhoto {
-    background: url("../../images/thinking.jpg");

```

Comment:
can't do webp images with fallback in css, so transitioned away from it where I could.